templates/wiki/base.html

```diff
@@ -0,0 +1,15 @@
+{% extends "wiki/base_site.html" %}
+{% load static %}
+
+{% block wiki_site_title %} - STACKn wiki{% endblock %}
```

Comment:
Good idea. This will of course need to be changed in several places for consistent naming. I have applied the change and we can discuss within the team.